File: C/Iniciante/1008/1008.c
#include <stdio.h>

int main(){
    int numero, horasTrabalhadas;
    float valorPorHora;

    scanf("%d %d %f", &numero, &horasTrabalhadas, &valorPorHora);

    printf("NUMBER = %d\n", numero);
    printf("SALARY = U$ %.2f\n", horasTrabalhadas*valorPorHora);

    return 0;
}

File: Python/1008.py
 # Problema 1079 - URI - Iniciante - Nível 1

#Leitura das variáveis
numero_funcionario = int(input())
horas_trabalhadas = int(input())
valor_hora = float(input())

#Cálculo do salário
salario = horas_trabalhadas * valor_hora

#Imprimindo as saídas
print(f'NUMBER = {numero_funcionario}')
print(f'SALARY = U$ {salario:.2f}')